docs/source/resources-algorithms.rst

```diff
@@ -3,4 +3,386 @@ OpenDCS Algorithms - Resources
 ###################################
 
 
-Content coming soon ...
+****************************
+Algorithms Resources - Recap
+****************************
+
+There are a number of "Standard" or "Baseline" algorithms that come 
+with OpenDCS when it is installed.  These can be thought of as banked 
+algorithms or functions that can be called upon when setting up a computation.
+These standard algorithms are generally basic arithmetic or time series
+manipulation or transformations.  For example, the average algorithm
+can be used to create a daily or monthly time series from an hourly or 
+daily time series, respectively.  In addition to simple arithmetic,
+or transformations, there are some set defined hydrologic specific
+algorithms that are commonly used in water management. For example
+there is an algorithm that can be called upon to calculate an incremental
+precipitation time series from a cumulative precipitation time series,
+and there are algorithms that can calculate flow based on various
+input time series.
+
+This page is intended to provide further details and examples of the 
+default algorithms.  Nearly all of these algorithms have properties
+that can be set to tweak the way the algorithm is executed. For example,
+the properties can address time shifts, missing values, change how default
+upper and lower bounds are handeled, etc.  It is recommended to 
+understand how the basic default algorithm behaves before advancing 
+to editing the properties.  
+
+This page is divided into three sections:
+
+#. OpenDCS Standard Algorithms
+#. OpenDCS Custom Python Algorithms
+#. OpenDCS Algorithm GUI Basics
+
+As a recap, below is a table of algorithms that come with OpenDCS installs.
+
```

Comment:
In the table below, some in the **Exec Class** column have .xml appended; I think the .xml should be dropped.

Reply:
Karl is correct, the Exec class should never end with xml or any other file extension.

Reply:
made table blank for "not visible" algorithms.  once tested will update table at later dates